packages/plugins/Traderxyz/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "tsBuildInfoFile": "dist/.tsbuildinfo"
+  },
+  "include": ["src", "src/**/*.json"],
+  "references": [
+    { "path": "../../theme" },
+    { "path": "../../mask/src" },
```

Comment:
You should not import from `mask/src`. What API you're going to use?

Reply:
Before some api availability in  mask shared, I was using it as source to get rid of rootDir error.

Let me fix and submit it.

Reply:
There is issue when i remove mask src as seen below
<img width="960" alt="image" src="https://user-images.githubusercontent.com/3579069/164972729-9711f043-c9cb-4985-b6a9-36349bb218d5.png">

Reply:
Yes. Because you should not import those files at first. I'm asking @guanbinrui if we should/could move it as a shared component.

Reply:
Thanks.

Reply:
Not resolved.

Reply:
Still not resolved.

Reply:
Because TokenTypePanel and TokenPanelType is not available in `@masknet/shared`

<img width="827" alt="image" src="https://user-images.githubusercontent.com/3579069/168591365-782655f0-cd7f-43c9-a4c6-036f19d8a5a5.png">